### PR TITLE
Added some links to contributing guide, fixed setup (and changed it to a bash script)

### LINKED ISSUE #222: src/fetch_datasets.py integrated into toolbox, but still mentioned in docs
The contribution guide still points to `src/fetch_datasets.py`, which was removed in 1604567f55c782874e9ee98f979f98cd3ab76d60. I'll be opening a pull request soon recreating the file, now containing just these two lines:

```python
from serenata_toolbox.datasets import fetch_latest_backup
fetch_latest_backup('data/')
```

This way, the `setup` file can be changed to a bash script, only invoking python to run this new file on the last lines.

### PR DESCRIPTION
**Pull request by cauebs**
Closes #222

### CODE REVIEW

File: setup [hunk 1]
```diff
@@ -1,21 +1,15 @@
...
-    print('The following executables are needed and were not found: unzip')
+echo "Downloading datasets (this might take several minutes depending on your internet connection)"
 
-print('Downloading datasets (this might take several minutes depending on your internet connection)')
-
-
-# This is imported here because serenata_toolbox is only installed when pip.main is called (line 9)
-from serenata_toolbox.datasets import fetch_latest_backup
-fetch_latest_backup('data/')
+python3 ./src/fetch_datasets.py
```
Comment (cuducos) [inline setup]: As the default Python for the environment is Python 3, I think we should use `python` instead of `python3`. What do you think?

Comment (cauebs) [inline setup]: I usually prefer to be explicit (because of weird distros like ubuntu), but in this case it shouldn't matter. Done!

File: setup [hunk 2]
```diff
@@ -1,21 +1,15 @@
-#!/usr/bin/env python3
+#!/usr/bin/bash
```
Comment (cuducos) [inline setup]: It doesn't seem to work here (macOS).

<details>

```console
$ ./setup
zsh: ./setup: bad interpreter: /usr/bin/bash: no such file or directory
$ which bash
/bin/bash
```

</details>


This leads me to another question: what about Windows users?

Maybe it's time to get rid of `setup` and replace it by a few command line instructions — so we can be more specific in addressing multiple platforms.

Comment (jtemporal) [inline setup]: > This leads me to another question: what about Windows users?

they do not have bash, it usually is cmd

> Maybe it's time to get rid of setup and replace it by a few command line instructions — so we can be more specific in addressing multiple platforms.

Couldn't agree more!

**Comment by cuducos** [comment]
@cauebs Considering [the last comments](https://github.com/datasciencebr/serenata-de-amor/pull/223#pullrequestreview-35783758) I think it is totally up to you:

* You can go ahead, `git rm setup` and add instructions to the `CONTRIBUTING.md` 
* Or just revert changes to `setup` and we handle that later

Besides that, thanks for the improvement in the documentation and to the port to the script that download the datasets ; )

**Comment by cuducos** [comment]
Closed by inactivity

**Pull request closed without merging**